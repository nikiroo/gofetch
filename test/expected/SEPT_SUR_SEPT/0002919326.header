0Les éleveurs de la zone infectée dispensés de cotisations sociales	null/SEPT_SUR_SEPT/0002919326		70
i  [Belgique] 26-09-2018 19:16
i
i    Les  exploitants  porcins  qui  se  trouvent  dans  la  zone de
i    sécurité  délimitée après l%27apparition, à la mi-septembre, du
i    virus de la peste porcine ...
i
