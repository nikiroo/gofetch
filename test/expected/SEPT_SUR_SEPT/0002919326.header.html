<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story-header'>
	<h2><a href='0002919326.html'>Les éleveurs de la zone infectée dispensés de cotisations sociales</a></h2>
	<div class='details'>([Belgique] 26-09-2018 19:16)</div>
	<br/>
	<div class='content' style='text-align: justify'>
		Les exploitants porcins qui se trouvent dans la zone de sécurité délimitée après l%27apparition, à la mi-septembre, du virus de la peste porcine ...
	</div>
<hr/>
</div>
</body>
